vendor/bat-native-ads/src/bat/ads/internal/ml/README.md

```diff
@@ -1,3 +1,19 @@
-# Machine Learning
+# Machine Learning (ML)
 
-Please add to it!
+This directory provides a consistent structure for implementing machine learning processes within Brave Core. The subdirectories are structured as:
+
+## Pipeline
+
+Defines the instructions for the various steps of a ML process. Often instructions are detailed by components loaded in from external Brave-sources.
+
+## Data
+
+Defines data structures useful for handling data when implementing ML processes
+
+## Transformation
+
+Algorithms for applying pre-processing transformations to model input data
+
+## Model
+
+Functions for applying ML models, such as generating predictions
```

Comment:
Great :-). For consistency could we end all sentences with a full-stop.

Reply:
As this file is in the root of ml/ it should have a table of contents linking to descriptions in each of those subdirectories.